game/test/contrib/components/InventoryComponentTest.java

```diff
@@ -203,7 +203,7 @@ public void getAllItemsInventoryNoAddedItemButCreated() {
                 new Item(
                         "Test item",
                         "Test description",
-                        new Animation("animation/missing_texture.png"));
+                        Animation.of("animation/missing_texture.png"));
```

Comment:
same

Reply:
Also Tests funktionieren eigentlich auch ohne jar. Und unter Windoof habe ich keine Probleme die Jars auszuführen. Die Methoden müssen dringend aufgeräumt werden. Ich glaube die Animationsklasse ist gerade so durcheinander, dass sie kaum nützlich ist. Animation.of(String,...) erstellt eine Singleframe Animation während Animation.of(FilePath,...) alle Frames sucht, die zu der Animation gehören. Der Ctor sorgt nur dafür, dass die Daten in einer passenden Datenstruktur liegen.